embassy-rp/Cargo.toml

```diff
@@ -88,6 +89,17 @@ boot2-w25x10cl = []
 ## ```
 boot2-none = []
 
+rp2040 = ["rp-pac/rp2040"]
+rp235x = ["rp-pac/rp235x"]
+rp235xa = ["rp235x"]
```

Comment:
the `rp235x` feature is "private", we don't want the user to enable it directly, only `rp235xa` or `rp235xb`. to make that clear can you prefix it with a `_`? (take a look at how features are structured in `embassy-nrf` for example)